vformer/models/classification/vanilla.py

```diff
@@ -8,21 +8,39 @@
 
 
 class VanillaViT(BaseClassificationModel):
+    """
```

Comment:
Please modify these docstrings by following the example.

Also, in every model definition class, let's include the name of the paper it implements and a link to it.
Eg. 
`Implmentation of "An Image is Worth 16x16 Words: Transformers for Image Recognition at Scale" https://arxiv.org/abs/2010.11929`